test/e2e/network/ingress.go

```diff
@@ -1114,17 +1114,23 @@ var _ = SIGDescribe("Ingress API", func() {
 		ginkgo.By("deleting")
 		err = ingClient.Delete(context.TODO(), createdIngress.Name, metav1.DeleteOptions{})
 		framework.ExpectNoError(err)
-		_, err = ingClient.Get(context.TODO(), createdIngress.Name, metav1.GetOptions{})
-		framework.ExpectEqual(apierrors.IsNotFound(err), true, fmt.Sprintf("expected 404, got %#v", err))
+		ing, err := ingClient.Get(context.TODO(), createdIngress.Name, metav1.GetOptions{})
+		framework.ExpectNoError(err)
```

Comment:
either of the following should be considered a success here:
1. apierrors.IsNotFound(err)
2. err == nil && ing.DeletionTimestamp != nil && len(ing.Finalizers) > 0

Reply:
Good call, done